tests-extended/dnsa_connection_pool.cpp

```diff
@@ -192,5 +192,46 @@ go_bandit([]() {
             std::this_thread::sleep_for(10ms);
             AssertThat(pool.poolState().available < min_spare_connections, IsTrue());
         });
+
+        it("can log dns error", [&]() {
+
+            auto& settings = getSettingsRef();
+            std::string hostname{"definitiely.not.resolvable.hostname"}, streamRedirectionResult{};
```

Comment:
streamRedirectionResult nepotrebuje `{}`

Reply:
Nepotrebuje, snazim se byt konzistentni. BTW, pouziti {} je preferovane podle  [cpp core guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es23-prefer-the--initializer-syntax)